reader/components/highlighter/utils/rects.js

```diff
@@ -26,13 +24,19 @@ export const groupRectsByPage = (rects, pagesRange, pdfViewer) => {
 
   // seems like it's already sorted but I couldn't verify it in API doc
   // https://developer.mozilla.org/en-US/docs/Web/API/Element/getClientRects
-  const sortedRects = sortBy(Array.from(rects), [r => r.top])
+  rects.sort((firstElement, secondElement) => {
+    if (firstElement.top < secondElement.top) return -1
+
+    if (firstElement.top > secondElement.top) return 1
+
+    return 0
+  })
```

Comment:
Here it will fit well:

```js
(a, b) => a.top - b.top
```

Reply:
I don't know why but I thought I need to return -1, 1 or 0 😄 Thanks

Reply:
https://github.com/oacore/reader/pull/104/commits/d4b1c329fc788693ce580fa9b7ec9cfe759d33a6